src/ethereum_test_fixtures/base.py

```diff
@@ -26,6 +55,41 @@ def output_base_dir_name(cls) -> str:
         """Return name of the subdirectory where this type of fixture should be dumped to."""
         return cls.fixture_format_name.replace("test", "tests")
 
+    @classmethod
+    def __pydantic_init_subclass__(cls, **kwargs):
```

Comment:
Wild :star_struck:

Reply:
This thing is like magic, I can't find the issue in the Pydantic repo that contains this example, but I think we can reuse this a lot with other types (like fork for example).